processor/deltatocumulativeprocessor/internal/metrics/ident.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"hash"
+	"hash/fnv"
+
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+type Ident struct {
```

Comment:
Why is this called `Ident`, while the `Metric` method returning it is called `Identity`?

Reply:
standardized on `Ident` in all places